lib/apiservers/engine/backends/archive.go

```diff
@@ -16,19 +16,21 @@ package backends
 
 import (
 	"archive/tar"
-	"context"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"os"
 	"path"
 	"strings"
 
+	"golang.org/x/net/context"
```

Comment:
The version of docker we are pinned to (1.13) still uses this context.  Newer version may use latest go context.